### PR TITLE
Update PMML readme

### PR DESCRIPTION
**Pull request by brianbelljr**
# This repository is public. Do not put here any private DataRobot or customer's data: code, datasets, model artifacts, .etc.

### CODE REVIEW

**Comment by yakov-g** [comment]
jenkins run test_integration_general test_integration_r_environment

File: model_templates/python3_pmml/README.md [hunk 1]
```diff
@@ -6,6 +6,9 @@ The supplied pmml file (iris_bin.pmml) is a pypmml model trained on [iris_binary
 with a Species as the target (classification), though any binary or regression model trained using the libraries
 outlined in [Python 3 pmml Drop-In Environment](../../public_dropin_environments/python3_pmml) will work.
 
+## NOTE
+You'll probably notice that the PMML file is not loaded explicitly in the custom.py file. Not to worry! In the drop-in environment we implicitly handle the score function for PMML so it is not needed, and therefore left out of the example. We handle certain other models types implicitly/automatically as well, like scikit-learn pickle files. 
+
```
Comment (yakov-g) [inline model_templates/python3_pmml/README.md:11]:
```suggestion
You can explicitly load an artifact by using `load_model` hook. More on hooksL [here](../../DEFINE-INFERENCE-MODEL.md#structured_inference_model)
```

**Review by yakov-g — APPROVED** [review]
@brianbelljr added a suggestion

**Comment by elatt** [comment]
Closing due to inactivity.

**Pull request closed without merging**